hazelcast-sql/src/main/java/com/hazelcast/sql/impl/type/converter/Converter.java

```diff
@@ -39,7 +42,7 @@
  * <p>
  * Java serialization is needed for Jet.
```

Comment:
add an explicit information that there should ever exist only 1 instance with given id. This is not guaranteed by the implementation - subclasses might break this. And currently we compare instances not ids.

Reply:
5e3f5be3

Reply:
✔️